omnimae/vision_transformer.py

```diff
@@ -0,0 +1,1121 @@
+#!/usr/bin/env python3
+# Portions Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Code modified from 
+# https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/vision_transformer.py ;
+# https://github.com/facebookresearch/deit/blob/main/models.py 
+# and https://github.com/facebookresearch/vissl/blob/main/vissl/models/trunks/vision_transformer.py
+# and is licensed under the license found in the
+# NOTICE file in the root directory of this source tree.
+
+
+# FIXME: DO NOT OPEN SOURCE THIS - we're missing proper attributions!
```

Comment:
need to remove this line

also, we need to copy any copyright headers from the sources we modified from